kogito-codegen/src/main/resources/class-templates/DecisionRestResourceTemplate.java

```diff
@@ -102,11 +94,18 @@ private Object extractSingletonDSIfSucceded(DMNResult result) {
             throw new DMNEvaluationErrorException(result);
         }
     }
-    
-    private void enrichResponseHeaders(DMNResult result) {
-        if (!result.getMessages().isEmpty()) {
+
+    private DMNResult enrichResponseHeaders(DMNResult result) {
```

Comment:
Why return a value that is not used?
```suggestion
    private void enrichResponseHeaders(DMNResult result) {
```

Reply:
It was a leftover from previous changes. Fixed now :+1: